tests/SprykerEcoTest/Zed/UnzerGui/codeception.yml

```diff
@@ -0,0 +1,41 @@
+namespace: SprykerEcoTest\Zed\UnzerGui
+
```

Comment:
tests/app/.gitkeep why do we need this file?

Reply:
yes. it is required for bootstrap inside Github Actions

Reply:
What is needed? tests/app/.gitkeep?